packages/gui/package.json

```diff
@@ -13,7 +13,8 @@
     "heroicons-react": "^1.0.15",
     "react": "experimental",
     "react-dom": "experimental",
-    "typescript": "3.9.3"
+    "typescript": "3.9.3",
+    "hasbin": "1.2.3"
```

Comment:
Nice! It’s a tiny nitpick, but can you move hasbin up so it’s all alphabetical?

Reply:
Sorted